guides/source/active_record_querying.md

```diff
@@ -680,6 +680,32 @@ SELECT * FROM customers WHERE (customers.last_name = 'Smith' OR customers.orders
 
 [`or`]: https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-or
 
+### AND Conditions
+
+`AND` conditions can be built by chaining `where` conditions.
```

Comment:
I'm not sure if this example needs to be added, but we don't want beginners to start using `and` where they should just use chained `where`s.